File: ungs-docsys-ms/src/main/java/com/ungs/docsys/dtos/JobApplicationApprovalResponseDto.java
package com.ungs.docsys.dtos;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class JobApplicationApprovalResponseDto {
    private Long id;
    private JobApplicationResponseDto jobApplication;
    private Integer[] usersApproval;
    private Boolean approved;
    private String reason;
    private AppUserResponseDto appUser;
    private LocalDateTime createdDate;
    private LocalDateTime updatedDate;
}

File: ungs-docsys-front/src/commons/dtos/job-application-approval-response.dto.ts
import { AppUserExtendedResponseDto } from "./app-user-extended-response.dto";
import { JobApplicationResponseDto } from "./job-application-response.dto";

export class JobApplicationApprovalResponseDto {
    id: number;
    jobApplication: JobApplicationResponseDto;
    usersApprovers : number[];    
    approved: boolean;
    reason: string;
    appUser: AppUserExtendedResponseDto;
    createdDate: Date;
    updatedDate: Date;
}